submissions/Roka20012/pop-up-menu/index.html

```diff
@@ -10,29 +11,91 @@
     <div class="container">
       <input class="check control" type="checkbox">
       <div class="mark"></div>
-      <input id="check-show-more" type="checkbox">
+      <input class="check-show-more" type="checkbox">
```

Comment:
You have to add `<label>` and connect him with your `<input type="checkbox">`. Also, You need to add `outline` to this` <label>`.

Reply:
Can you explain why we need it?)
It also works without it

Reply:
`label` needs for accessibility.
`outline` on focused element indicates to user what element in focus.
`.visually-hidden` visually hides the item but it is available for screen readers.  Also, if you change `opacity`, the current element becomes transparent(and you need out image under checkbox using `z-index`) and still takes its place. This class cuts out so that it does not take (free) space.

Reply:
Thank you for your explanation.
I think , i fix it :)